src/app/components/listing/details/criteria/criteria.jsx

```diff
@@ -83,36 +93,39 @@ function ChplCriteria(props) {
         ))}
       { (criteria.filter((cc) => cc.criterion.removed && (cc.success || props.viewAll)).length > 0)
         && (
-          <Accordion
-            className={classes.NestedAccordionLevelOne}
-          >
-            <AccordionSummary
-              className={classes.NestedAccordionLevelOneSummary}
-              expandIcon={<ExpandMoreIcon color="primary" fontSize="large" />}
-              id="removed-header"
+          <div>
```

Comment:
Is this element doing anything?

Reply:
I read some documentation that if you wrap the accordion in a div, you wont get the awkward :before line that shows. Same as the other div below.